lib/views/query-error-tile.js

```diff
@@ -10,6 +10,7 @@ export default class QueryErrorTile extends React.Component {
         status: PropTypes.number.isRequired,
```

Comment:
Short answer: not yet.

We don't actually have any uniform way to refresh the GitHub tab during _normal_ operations 😅 . It actually refreshes when you fetch or push, but that reruns all of the QueryRenderers entirely, which doesn't confuse the Accumulators (I did actually test this out to verify). If we nest this in a refetch container when we change the refresh behavior (maybe with #1934?) then yeah we'll need to pass in a well-behaved `onDidRefetch()` method. Once we do that, it might make sense to integrate some auto-retry on network change behavior here or a retry button - it depends on the UI we settle on.